### ISSUE TITLE
where line starts with a brackets there is no validation for a long line connection

### ISSUE THREAD

**Issue by yaron-shamul** [PyCQA/pylint#5490]
### Bug description

I used to parse some json object and the brackets was very long so I made it to continue with another line, and I got this warning.
![image](https://user-images.githubusercontent.com/75207401/145025889-52f195aa-48e7-4935-b1e9-b14085a72c59.png)
![image](https://user-images.githubusercontent.com/75207401/145025929-7e0d3900-2eb2-4e13-9e5f-b68c06059466.png)

### Command used

```shell
py -m pylint playlist.py
```

### Pylint output

```shell
playlist.py:195:12: W0104: Statement seems to have no effect (pointless-statement)
playlist.py:197:12: W0104: Statement seems to have no effect (pointless-statement)
```

### Expected behavior

to not write any warning about this line

### Pylint version

```shell
pylint 2.12.1
astroid 2.9.0
Python 3.8.6 (tags/v3.8.6:db45529, Sep 23 2020, 15:52:53) [MSC v.1927 64 bit (AMD64)]
```

### OS / Environment

windows 10

**Comment by Pierre-Sassoulas** [comment]
Hello, thank you for opening the issue. Could you provide the code as text instead of as an image ? Text is crawl-able by search engines, readable by visually impaired persons and copy pastable to test the issue fast, while images are not.

Also, I don't think your code is doing what you think it's doing. There is two separated lines here, you can't made a line "continue with another line" by just creating another line, you need a backslash for that.

**Comment by yaron-shamul** [comment]
I checked again the lint of lines with a backslashes and you're right it's worked as expected, thank you!
